3p/ampcontext.js

```diff
@@ -312,8 +313,22 @@ export class AbstractAmpContext {
       this.setupMetadata_(this.win_.name);
     }
   }
-}
 
+  /**
+   * Send 3p error to parent iframe
+   * @private
+   */
+  report3pError_() {
+    this.win_.onerror = message => {
+      if (message) {
+        this.client_.sendMessage(MessageType.USER_ERROR_IN_IRAME, dict({
+          'message': message,
```

Comment:
Don't you want the stack?

Reply:
I think I only need the message to be passed through

Reply:
The 4th argument (I think) of `onerror` is the `Error` object or undefined.

So you can do `'stack': error && error.stack`